lib/mongo_mapper/plugins.rb

```diff
@@ -14,9 +22,18 @@ def plugin(mod)
       plugins << mod
     end
 
+    def direct_descendants
+      Plugins.direct_descendants_hash[self]
+    end
```

Comment:
Given Ruby 3.0 and ActiveSupport 7.0, ActiveSupport::DescendantsTracker#direct_descendants doesn't work with Modules. So let's implement it by ourselves.

Reply:
Looks good